integration/integration_test.go

```diff
@@ -462,6 +465,9 @@ func prepare(t testing.TB) {
 		remote = true
 	case "sql":
 		cfg.DatabasePath = "postgres://localhost/cayley_test"
```

Comment:
CockroachDB is the same as postgres and mysql has no protocol part in URL. But we can define them ourselves, so they are more consistent.
I'm okay with flavor option, since I can copy address directly from other SQL packages for Go. But not a big deal.